lib/actions/EventLambdaSchedule.js

```diff
@@ -45,73 +45,95 @@ module.exports = function (S) {
 
       let func           = S.getProject().getFunction( _this.evt.options.funcName ),
           event          = func.getEvent( _this.evt.options.name ),
-          populatedEvent = event.toObjectPopulated({stage: _this.evt.options.stage, region: _this.evt.options.region}),
+          stage          = _this.evt.options.stage,
+          region         = _this.evt.options.region,
+          populatedEvent = event.toObjectPopulated({ stage, region }),
           functionName   = event.getFunction().getDeployedName(_this.evt.options),
-          ruleName       = functionName + '-' + populatedEvent.name + '-' +  _this.evt.options.stage,
-          awsAccountId   = _this.aws.getAccountId(_this.evt.options.stage, _this.evt.options.region),
-          lambdaArn      = 'arn:aws:lambda:' + _this.evt.options.region + ':' + awsAccountId + ':function:' + functionName,
-          stage          = _this.evt.options.stage;
+          ruleName       = S.getProject().getName() + '-' + populatedEvent.name,
+          awsAccountId   = _this.aws.getAccountId(stage, region),
+          lambdaArn      = 'arn:aws:lambda:' + region + ':' + awsAccountId + ':function:' + functionName;
 
 
-      populatedEvent.config.enabled = populatedEvent.config.enabled ? 'ENABLED' : 'DISABLED';
-
       SUtils.sDebug(`Putting CloudWatchEvents Rule ${ruleName}`);
 
+      // The rule itself is always enabled. However, the triggers might not
+      console.log("_this.evt.options = ", _this.evt.options);
       var params = {
         Name: ruleName,
+        Description: _this.evt.options.description,
         ScheduleExpression: populatedEvent.config.schedule,
-        State: populatedEvent.config.enabled
+        State: 'ENABLED'
```

Comment:
Shouldn't we pull the `enabled` setting from the event config? I suppose with shared events we should default to not changing it if it's disabled on the server, but if we're deploying for the first time and the config is set to `disabled` this would be unexpected behavior.